scripts/azureml-assets/azureml/assets/deployment_config.py

```diff
@@ -201,8 +201,8 @@ def should_create(self, asset_type: assets.AssetType, asset_name: str) -> bool:
 class TagsSchema(Schema):
     """Tags schema."""
 
-    add = fields.Dict(fields.Str(), fields.Str())
-    replace = fields.Dict(fields.Str(), fields.Str())
+    add = fields.Dict(fields.Str())
+    replace = fields.Dict(fields.Str())

```

Comment:
Please revert this. Tag values are strings. If any other datatypes need to be handled, conversion should be upstream.

Reply:
@elesel tag values supports list and dict now as well for all asset types

Reply:
this was probably updated very recently

Reply:
okay I found one issue. Though you can push a list, but a translation to string happens and eventually it's a list. Updating